waspc/data/Generator/templates/sdk/wasp/server/auth/email/utils.ts

```diff
@@ -64,6 +64,11 @@ async function sendEmailAndSaveMetadata(
   // so the user can't send multiple requests while the email is being sent.
   const providerId = createProviderId("email", email);
   const authIdentity = await findAuthIdentity(providerId);
+
+  if (!authIdentity) {
+    throw new Error(`User with email: ${email} not found.`);
+  }
```

Comment:
Yep, we had a bug here 😄

Reply:
The failure scenario here was: "User doesn't exist -> trying using the user object" and then this would fail with a nondescript error "Accessing ... from undefined". 

This failure scenario is now explicit and fails with "User with email X not found".

I don' think is a breaking change for our users, it's an internal method.